drivers/clock_control/clock_control_wch_rcc.c

```diff
@@ -154,6 +154,7 @@ static int clock_control_wch_rcc_init(const struct device *dev)
 			RCC->CFGR0 |= RCC_PLLSRC;
 		} else if (IS_ENABLED(WCH_RCC_PLL_SRC_IS_HSI)) {
 			RCC->CFGR0 &= ~RCC_PLLSRC;
+			EXTEN->EXTEN_CTR |= EXTEN_PLL_HSI_PRE;
```

Comment:
This doesn't exist on the ch32v003, so add a `#if defined(EXTEN_PLL_HSI_PRE)` around it.